script.game.whatthemovie/addon.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.game.whatthemovie" name="WhatTheMovie" version="1.0.9" provider-name="Tristan Fischer (sphere@dersphere.de)|tadi|skiller2k1">
+<addon id="script.game.whatthemovie" name="WhatTheMovie" version="1.0.10" provider-name="Tristan Fischer (sphere@dersphere.de)|tadi|skiller2k1">
     <requires>
-        <import addon="xbmc.python" version="2.1.0"/>
+        <import addon="xbmc.python" version="2.14.0"/>
         <import addon="script.module.beautifulsoup" version="3.0.8"/>
     </requires>
-    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.python.script" library="default.py">
+        <provides>game</provides>
```

Comment:
Does this work on older Kodi/XBMC versions?

Reply:
Hey @Rechi. Yes it installs and works fine under Gotham. Game is just ignored and the addon is placed under programs.

I get an error regarding the translation stuff though:
```
10:33:59 T:2962710528  NOTICE: -->Python Interpreter Initialized<--
10:33:59 T:2962710528  NOTICE: [ADDON][WhatTheMovie] Version 1.0.10 started
10:33:59 T:58559040   ERROR: POParser: unable to read PO file header from file: /Users/me/Library/Application Support/XBMC/addons/script.game.whatthemovie/resources/language/English/strings.po
10:33:59 T:2962710528   ERROR: Previous line repeats 6 times.
10:33:59 T:2962710528  NOTICE: [ADDON][WhatTheMovie] Error:  - not all arguments converted during string formatting
10:33:59 T:2962710528  NOTICE: [ADDON][WhatTheMovie] Traceback: ['Traceback (most recent call last):\n', '  File "/Users/me/Library/Application Support/XBMC/addons/script.game.whatthemovie/resources/lib/gui.py", line 189, in onInit\n    self.login()\n', '  File "/Users/me/Library/Application Support/XBMC/addons/script.game.whatthemovie/resources/lib/gui.py", line 665, in login\n    self._showUserScore(self.score)\n', '  File "/Users/me/Library/Application Support/XBMC/addons/script.game.whatthemovie/resources/lib/gui.py", line 496, in _showUserScore\n    % (score[\'ff_score\'], score[\'all_score\']))\n', 'TypeError: not all arguments converted during string formatting\n']
10:34:13 T:2964307968 WARNING: DARWINOSX: underflow (0 vs 4096 bytes)
```

Reply:
@da-anda looks like you miss the header on the strings.po file